net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/BGP.xml

```diff
@@ -37,10 +37,9 @@
                                 <default>1</default>
                                 <Required>Y</Required>
                         </enabled>
-                        <address type="TextField">
+                        <address type="HostnameField">
```

Comment:
should this support hostnames too?

Reply:
Normally not, but HostnameField is ATM the only fieldtype validating v6 addresses. Didn't found a good mask to validate v4 and v6 so I asked @AdSchellevis in another project for this type. Don't know if he also likes to introduce a new fieldtype for v4 and v6 only, or you have an idea for stable masking v4 and v6?

Reply:
How about NetworkField?
That is v4 and v6 + optional CIDR